docs/about.html

```diff
@@ -0,0 +1,132 @@
+            </div>
+          </div>
+          <div class="p-5">
+              <img class="img-fluid" src="images/pearplanner.jpg" alt="Image of the CEG3120 Class, WSU, Fall 2017">
+            </div>
+        </div>
+    </section>
+
+	<!-- Text Right, Image Left -->
+    <section>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6">
+            <div class="p-5">
+              <img id="imgBorder" class="img-fluid rounded-circle" src="images/classPicture.jpg" alt="Image of students from Mr. Sanchez' CEG-3120 Fall 2017 Class">
+            </div>
+          </div>
+          <div class="col-md-6">
+            <div class="p-5">
+              <h2 class="display-4">Key Features</h2>
+              <p>The current implementation of the RaiderPlanner, formerly the PearPlanner has the 4 central capabilities from the original project still at its core, with the implementation of several changes made by the CEG-3120 students. This planner is one that can be used by a student to keep track of the major items due over the course of the semester. Tools that allow the user to add calendar reminders, make notes and assign tasks for each assignment due.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  
+	<!-- Text Left, Image Right -->
+    <section>
+	  
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6 order-2">
+            <div class="p-5">
+			  <!-- Image retrieved from: http://businessessays.net/operation-management/project-planning-techniques-cpa-pert-and-gantt-charts/ -->
+              <img id="imgBorder" class="img-fluid rounded-circle" src="images/movingforward.jpeg" alt="Planning Image">
+            </div>
+          </div>
+          <div class="col-md-6 order-1">
+            <div class="p-5">
+			  <h2 class="display-4">RaiderPlanner:<br>The Way Ahead</h2>
+              <p>The Fall 2017 students of Mr. Sanchez' CEG-3120 class started out the semester working on several issues identified by Mr. Sanchez. As part of the course progression, students were asked to identify issues, or enhancements for the RaiderPlanner. The plan is to incorporate some or all of these recommended changes through the course of the project.<br><br>To find out more about the identified issues associated with this project, click <a href="https://github.com/rsanchez-wsu/RaiderPlanner/issues" target="_blank">HERE</a>.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Footer -->
+    <footer class="py-3 bg-dark">
+      <div class="footer">
+        <p> Would you like to contribute to this project? Click <a href="contribute.html">HERE</a> to find out more information </p>
+        <p class="m-0 text-center text-white">Copyright &copy; RaiderPlanner 2017</p>
```

Comment:
RaiderPlanner is not an organization, so this should rather say "RaiderPlanner Contributors" and then link to the AUTHORS file in the project repository.  I have just now added the file.